resources/css/humhub.wiki.less

```
@@ -1056,3 +1056,5 @@
     padding: 0;
   }
 }
+/* CSS Styling for toggle button */
+.toggle-numbering {margin-left: auto;padding-right: 10px;}
```

Please stick to code style.

```suggestion
/* CSS Styling for toggle button */
.toggle-numbering {
  margin-left: auto;
  padding-right: 10px;
}
```